app/views/teams/_form.html.slim

```diff
@@ -69,6 +69,14 @@
               = s.link_to_remove 'Remove', class: 'btn btn-default'
       .form-group.form-btn-group
         = f.link_to_add 'Add another source', :sources, class: 'btn btn-primary col-sm-offset-3'
+  -if can?(:update_conferences, @team)
```

Comment:
This is probably personal preference, but I find "conferences" here a bit misleading, since the form is actually about "attendances" to conferences... I'd also suggest to more stick to the pattern of looking at the single user's abilities rather then the team - it's a in the end a single user who submits the form 😉 

Do you have a better idea for expressing this?